backend/configuration/configure_appwrite.py

```diff
@@ -0,0 +1,539 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2021 Felix Steinkohl <steinkohl@campus.tu-berlin.de>
+
+import os
+from dataclasses import dataclass
+import pandas as pd
+from appwrite.client import Client
+from appwrite.services.users import Users
+from appwrite.services.teams import Teams
+from appwrite.services.database import Database
+from appwrite.exception import AppwriteException
+
+
+@dataclass
+class UserData:
+    """
+    Dataclass for specifying user data
+    """
+
+    name: str
+    email: str
+    team: str
+    roles: [str]
+    init_password: str
+
+
+def load_parameter(file: str) -> pd.DataFrame:
+    """
+    Reads the given file and returns 'Parameter' sheet as a Dataframe.
+
+    :param file: path of file containing the parameter data that shall be loaded
+    :type file: str
+    :return: Data of the 'Parameter' sheet
+    :rtype: pd.DataFrame
+    """
+    ending = str(file).lower().split(".")[-1]
+    if ending in ["xlsx", "xls"]:
+        try:
+            return pd.read_excel(file, dtype=str, sheet_name="Parameter")
+        except ValueError:
+            # no 'Parameter' in file
+            return pd.DataFrame()
+    else:
+        raise TypeError(
+            "Only xlsx format is supported if parameters have to be loaded from file!"
+        )
+
+
+def set_environment_variables(file: str):
+    """
+    Loads excel file and writes given parameters as environment variables.
+
+    :param file: str
+    :return: None
+    """
+    try:
+        df = load_parameter(file)
```

Comment:
'df' is a standard name for pandas DataFrames. I agree, it could be 'parameter_df' but it's only usage is the line below, so i think it's pretty obvious what it is for, isn't it?

Reply:
Only if you read bottom up and it's still not clear what `df` contains (and therefore hard to check whether it fits the intension).
Also, this deliberately violates our code review guidelines which state to use pronounceable variables where this is not the case.

In between you have to remember the definition because the meaning doesn't derive from the name, which increases cognitive load.

It would be a mislead "standard" and I can well imagine some school kids naming their python vars like this. It's like calling a variable `var`, `container`, `list`, `array`, `data`, ... It transfers no meaning.